android/expoview/src/main/java/host/exp/exponent/ReactNativeStaticHelpers.kt

```diff
@@ -0,0 +1,184 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+package host.exp.exponent
+
+import android.util.Log
+import com.facebook.proguard.annotations.DoNotStrip
+import com.facebook.react.common.JavascriptException
+import host.exp.exponent.network.ExponentNetwork
+import host.exp.expoview.Exponent
+import okhttp3.CookieJar
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+@DoNotStrip
+object ReactNativeStaticHelpers {
+  private val TAG = ReactNativeStaticHelpers::class.java.simpleName
+
+  private var bundleUrl: String? = null
+  @DoNotStrip
+  @JvmStatic fun setBundleUrl(bundleUrl: String?) {
+    this.bundleUrl = bundleUrl
+  }
+
+  @DoNotStrip
+  @JvmStatic fun reloadFromManifest(activityId: Int) {
+    try {
+      Class.forName("host.exp.exponent.kernel.Kernel")
+        .getMethod("reloadVisibleExperience", Int::class.javaPrimitiveType)
+        .invoke(null, activityId)
+    } catch (e: Exception) {
+      Log.e("reloadFromManifest", "Unable to reload visible experience", e)
+    }
+  }
+
+  @DoNotStrip
+  @JvmStatic fun getBundleUrlForActivityId(
+    activityId: Int,
+    mainModuleId: String?,
+    bundleTypeId: String?,
+    host: String?,
+    devMode: Boolean,
+    jsMinify: Boolean
+  ): String? {
+    return try {
+      // TODO(wschurman): the argument order here looks out of order
```

Comment:
good catch. also Kernel has old implementations of this method we should be able to delete.